src/components/sbb-action-group/sbb-action-group.scss

```diff
@@ -0,0 +1,77 @@
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include host-component-properties;
+
+$breakpoints: 'zero', 'micro', 'small', 'medium', 'large', 'wide', 'ultra';
+$vertical-orientations: (
+  start: flex-start,
+  center: center,
+  stretch: stretch,
+  end: flex-end,
+);
+$horizontal-orientations: (
+  start: flex-start,
+  center: center,
+  stretch: space-between,
+  end: flex-end,
+);
+
+:host([orientation='horizontal']) {
+  --sbb-action-group-orientation: row;
+  --sbb-action-group-align-items: center;
+}
+
+@each $key, $value in $horizontal-orientations {
+  /* stylelint-disable-next-line plugin/stylelint-bem-namics */
+  :host([orientation='horizontal'][align-group='#{$key}']) {
+    --sbb-action-group-justify-content: #{$value};
+  }
+}
+
+:host([orientation='vertical']) {
+  --sbb-action-group-orientation: column;
+}
+
+@each $key, $value in $vertical-orientations {
+  /* stylelint-disable-next-line plugin/stylelint-bem-namics */
+  :host([orientation='vertical'][align-group='#{$key}']) {
+    --sbb-action-group-align-items: #{$value};
+  }
+}
+
+@each $breakpoint in $breakpoints {
+  @include mq($from: #{$breakpoint}) {
+    @each $key, $value in $horizontal-orientations {
+      // horizontal-from overrides orientation vertical
+      :host([orientation='vertical'][horizontal-from='#{$breakpoint}'][align-group='#{$key}']) {
+        --sbb-action-group-orientation: row;
+        --sbb-action-group-align-items: center;
+        --sbb-action-group-justify-content: #{$value};
+      }
+    }
+  }
+}
+
+.sbb-action-group {
+  display: flex;
+  flex-direction: var(--sbb-action-group-orientation);
+  gap: var(--sbb-spacing-fixed-4x);
+  justify-content: var(--sbb-action-group-justify-content);
+  align-items: var(--sbb-action-group-align-items);
+
+  ::slotted([align-self='start']) {
+    margin-inline-end: auto;
```

Comment:
Why is not align-self used here?

Reply:
From what I remember, align-self moves the elements in the cross axis; in this case we want to move them along the main axis.